src/ipyaladin/widget.py

```diff
@@ -161,6 +166,35 @@ def _handle_custom_message(self, _: any, message: dict, __: any) -> None:
         elif event_type == "select" and "select" in self.listener_callback:
             self.listener_callback["select"](message_content)
 
+    @property
+    def wcs(self) -> WCS:
+        """The world coordinate system of the Aladin Lite widget.
+
+        Returns
+        -------
+        WCS
+            An astropy WCS object representing the world coordinate system.
+
+        """
+        if "RADECSYS" in self._wcs:  # RADECSYS keyword is deprecated for astropy.WCS
+            self._wcs["RADESYS"] = self._wcs.pop("RADECSYS")
+        return WCS(self._wcs)
+
+    @property
+    def fov_xy(self) -> Tuple[Angle, Angle]:
+        """The field of view of the Aladin Lite along the two axis.
```

Comment:
```suggestion
        """The field of view of the Aladin Lite along the two axes.
```

(tricky plural form)